inference-engine/samples/benchmark_app/remotecontext_helper.hpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+#include <vector>
+#include <map>
+#include <memory>
+#include <inference_engine.hpp>
+#include "infer_request_wrap.hpp"
+
+class RemoteContextHelper {
```

Comment:
Please add the documentation to this class

Reply:
Updated.